package.json

```diff
@@ -93,7 +93,8 @@
     "uint8array-extras": "^0.5.0",
     "utm": "^1.1.1",
     "validate-color": "^2.2.4",
-    "zustand": "^4.4.6"
+    "zustand": "^4.4.6",
+    "expo-splash-screen": "~0.26.5"
```

Comment:
move this so that's placed in the file based on alphabetical order (default when running `npm install ...`)

Reply:
To be honest, I don't know why, but when I installed that package, it was placed last in the list.